modules/contentbox/modules/contentbox-ui/interceptors/CBRequest.cfc

```diff
@@ -107,17 +107,24 @@ component extends="coldbox.system.Interceptor" {
 			}
 		}
 
+		var linkLogout = "#prc.cbAdminEntryPoint#/security/doLogout";
+
 		// Render the admin bar out
 		var adminBar = renderView(
 			view   = "adminbar/index",
 			module = "contentbox-ui",
 			args   = {
 				oContent       : oContent ?: javacast( "null", "" ),
 				linkEdit       : linkEdit,
+				linkLogout     : linkLogout,
```

Comment:
Since the variables `linkLogout` is only used once, don't declare it, just add the value here inline

Reply:
Done.